### DISCUSSION TITLE
Are there any plans to support other installation methods?

### DISCUSSION THREAD

**Discussion by dminca** [PatrickF1/fzf.fish#250] [Ideas]
Hi there,

I love this plugin and I bumped [into a blog](https://travisbrady.github.io/posts/moving-to-fish-shell/) post which basically says _Fisher is necessary to install fzf.fish_ .

That's also the case for me currently, I've just installed initially `omf` and now have to install a secondary plug-in manager just to install `fzf.fish`, I think I'm not the only one in this situation.

Are there any plans to support installation via `omf` plug-in manager or even `homebrew` ?

Thank you

**Comment by PatrickF1** [comment]
Hi @dminca, fisher is not necessary to install fzf.fish. You can install it just fine with omf and everything will work. The only thing is that the uninstall method will only take you 90% of the way--you'll have to close your shell after uninstalling to finish uninstallation. That's it.

I should contact Travis Brady to let him he doesn't have to install fisher.

**Reply by dminca** [reply]
Oh I didn't know that. Thanks for the quick reply Patrick and for the follow-up action 😄 

I don't know if this should be specified in the `README.md` in case other people may raise this question.

**Reply by PatrickF1** [reply]
It already says
`fzf.fish can be installed manually or with other plugin managers but only Fisher is officially supported.`
how can I make this more clear?

**Reply by dminca** [reply]
A TOC to the readme would help improve that.

In this case, it was my fault, must've missed that part on a quick-skim

**Reply by PatrickF1** [reply]
Gotcha, the TOC is on my todo list but I haven't done it yet cause it makes the readme file even longer and I want to keep it short.

Btw, linking the background issues re: OMF here. https://github.com/PatrickF1/fzf.fish/pull/59 and https://github.com/PatrickF1/fzf.fish/pull/58
If you read them, you'll see it's such as small incompatibility it shouldn't matter.
